src/chat/composer.json

```diff
@@ -30,7 +30,7 @@
     },
     "require-dev": {
         "ext-redis": "*",
-        "doctrine/dbal": "^3.3|^4.0",
+        "doctrine/dbal": "^4.0",
```

Comment:
I'm not sure that's a good idea. We're going to recommend DBAL v3 for the time being in Symfony.
See https://github.com/symfony/symfony/issues/62481 and related
What's the motivation for the bump? We tend to favor wide ranges in the Symfony project, unless reasons.

Reply:
Just cleanup, so let's keep v3 support